### PR TITLE
updating typo + idea form

### PR DESCRIPTION
**Pull request by leomordasini**

### CODE REVIEW

**Review by tenatzen — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/Idea_submission.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+name: Submit your idea proposal
+description: Submit your idea proposal
+title: "[Idea]: "
+labels: ["idea"]
+assignees:
+  - test
```
Comment (psmulovics) [inline .github/ISSUE_TEMPLATE/Idea_submission.yml:6]: We need actual username here

**Review by psmulovics — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/Idea_submission.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    id: other
+    attributes:
+      label: Any other key information
+      description: Please add any other key information about your idea here
+      render: shell
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [FINOS Code of Conduct](https://example.com)
```
Comment (psmulovics) [inline .github/ISSUE_TEMPLATE/Idea_submission.yml:67]: We need to fill out the url

**Review by psmulovics — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/Idea_submission.yml (hunk 1 above)
Comment (leomordasini) [reply .github/ISSUE_TEMPLATE/Idea_submission.yml]: DONE.

File: .github/ISSUE_TEMPLATE/Idea_submission.yml (hunk 2 above)
Comment (leomordasini) [reply .github/ISSUE_TEMPLATE/Idea_submission.yml]: DONE.

**Review by psmulovics — APPROVED** [review]

**Pull request merged**